AirCasting/APICommunicator/StreamDownloader.swift

```diff
@@ -0,0 +1,74 @@
+// Created by Lunar on 29/03/2022.
+//
+
+import Foundation
+import Resolver
+
+struct StreamWithMeasurementsDownstream: Decodable {
+    let title: String
+    let username: String
+    let measurements: [StreamWithMeasurementsDownstream.Measurements]
+    let id: Int
+    let lastMeasurementValue: Double
+    let sensorName: String
+    let sensorUnit: String
+    
+    struct Measurements: Decodable {
+        let value: Double
+        let time: Int
+        let longitude: Double
+        let latitude: Double
+    }
+}
+
+protocol StreamDownloader {
```

Comment:
`StreamDownstream` here would be appropriate (although funny naming coincidence)